### PR TITLE
Flashpoint Release 201

### PR DESCRIPTION
**Pull request by crestdatasystems**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Description
- Fixed an issue with the resource URL incident field in cases where it is not populated by default.

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

File: Packs/Flashpoint/Integrations/Ignite/Ignite.py [hunk 1]
```diff
@@ -142,7 +142,7 @@
 ALERT_RESOURCE_URL = {
     'communities': '/search/context/communities/{}',
     'marketplaces': '/search/context/marketplaces/{}',
-    'media': '/search/results/media?include.media_id={}',
+    'media': '/search/results/media?include.date=all+time&include.media_id={}',
```
Comment (tcarmeli1) [inline Packs/Flashpoint/Integrations/Ignite/Ignite.py:145]: why did you changed the the original url?

**Review by tcarmeli1 — CHANGES_REQUESTED** [review]
Hi @crestdatasystems ,
Thank you for your contribution!
Good work :)
Please see my comments.
General comment- Please add new unit tests for the new functionality you added.
Please feel free to reach out to me with any questions - I'm available here or on slack :)
Thanks again

File: Packs/Flashpoint/Integrations/Ignite/Ignite.py (hunk 1 above)
Comment (crestdatasystems) [reply Packs/Flashpoint/Integrations/Ignite/Ignite.py:145]: Hi @tcarmeli1, The old `media` based URL was not having the parameter to search it on bases of all time. So, we have added date parameter to resolve the issue.

**Comment by crestdatasystems** [comment]
> Hi @crestdatasystems , Thank you for your contribution! Good work :) Please see my comments. General comment- Please add new unit tests for the new functionality you added. Please feel free to reach out to me with any questions - I'm available here or on slack :) Thanks again

Hi @tcarmeli1,
 Thanks for reviewing the PR.
 The intention of this release was to resolve the issue of unavailability of the Resource URL.
-  In some of type of alerts, the Resource URL is directly available from the API response.
- To resolve the issue, we have forged custom Resource URL based on the type of Alert.
- So, there is no change in any major functionality. And we also wanted to check that the existing test cases should pass, and there should be only change in test data to check that existing functionality haven't affected due to this release.
- This was the reason to not modify the test cases.

File: Packs/Flashpoint/Integrations/Ignite/Ignite.py (hunk 1 above)
Comment (tcarmeli1) [reply Packs/Flashpoint/Integrations/Ignite/Ignite.py:145]: So it will not lead to backwards compatibility right?

**Comment by tcarmeli1** [comment]
Hi @crestdatasystems ,
Thank you for your response!
It depends on your decision but i would suggests adding one more test in order to test the functionality when a Resource URL is not provided by the API.
In addition, I would appreciate if you can please create a Demo video so i can go on and merge this pr, we can also conduct a Demo meeting if you prefer.
Thank you so much!

File: Packs/Flashpoint/Integrations/Ignite/Ignite.py (hunk 1 above)
Comment (crestdatasystems) [reply Packs/Flashpoint/Integrations/Ignite/Ignite.py:145]: The change doesn't affect any functionality. Its only related to the platform side evaluation of the parameters.

**Review by tcarmeli1 — APPROVED** [review]
Watched the demo- all looks great.
Great work!

**Pull request merged**